Dockerfile

```diff
@@ -1,6 +1,6 @@
-ARG infra_image=newrelic/infrastructure:latest
+ARG infra_image=newrelic/infrastructure-bundle
 
-FROM golang:1.10 as builder
+FROM golang:1.14 as builder
```

Comment:
I guess we centos5 is out of the scope :trollface:

Reply:
Really? We cannot upgrade Go because of CentOS 5?

Reply:
don't worry, it doesn't apply to this integration